tools/ui-components/utils/gen-component-template.ts

```diff
@@ -4,40 +4,52 @@ import React from 'react';
 
 import { ${name}Props } from './types';
 
-export const ${name} = ({}: ${name}Props) => {
-  return <div>Hello, I am a ${name} component</div>;
+export const ${name} = (props: ${name}Props):JSX.Element => {
+  return <div {...props}>Hello, I am a ${name} component</div>;
 };
 `;
 
 // types.ts
 export const type = (name: string): string => `
 export interface ${name}Props {
-  className?: string
+  className?: string;
+  children?: string;
 }
 `;
 
 // component.test.tsx
 export const test = (name: string): string => `
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
+// import userEvent from '@testing-library/user-event';
 
 import { ${name} } from '.';
 
 describe('<${name} />', () => {
-  it('should render correctly', () => {});
+  it('should render ${name} correctly', () => {
+    render(<${name}>Test</${name}>);
+    expect(screen.getByText('Test')).toBeInTheDocument();
+  });
```

Comment:
stop the annoying error, that stops the early commits

Reply:
I will think of way to have both, I think it's possible to create simple test that check for class value, but I need to understand the library more